lib/hook.js

```diff
@@ -46,6 +46,22 @@ function Hook() {
 	};
 }
 
+function findInDir (dir, include = /\.js$/, exclude = /^[._]/, fileList = []) {
+	const files = fs.readdirSync(dir);
+	files.forEach((file) => {
```

Comment:
Generally, you should use a for-of loop instead of a `forEach()`. for-of is much faster and easier to read.

Secondly, if an arrow function only has 1 argument, no need for parenthesis.

Reply:
changed to for-of loop